### PR TITLE
Feature progress step scroll view props

### PR DESCRIPTION
**Pull request by DanSmit**

### CODE REVIEW

**Comment by DanSmit** [comment]
@colbymillerdev any thoughts on this?

**Comment by colbymillerdev** [comment]
Hey Dan - Sorry for the delay, I'm just getting back from vacation. I'll try to look this over tomorrow and get back to you. Thanks for the PR!

**Comment by colbymillerdev** [comment]
@DanSmit This change looks good! I like the idea of allowing any props to be passed in. Just FYI for any future PRs, try not to reformat the entire file if possible.

I'll get this merged in tomorrow once I have a chance to pull down the changes and quickly test it out.

**Comment by DanSmit** [comment]
@colbymillerdev apologies for the reformatting. Will keep it in mind for the next PR

File: examples/ExampleOne.js [hunk 1]
```diff
@@ -4,74 +4,82 @@ import { View, Text } from 'react-native';
...
+
+	render() {
+		return (
+			<View style={{ flex: 1, marginTop: 50 }}>
+				<ProgressSteps>
+					<ProgressStep
+						label="Payment"
+						onNext={this.onPaymentStepComplete}
+						onPrevious={this.onPrevStep}
+						scrollViewProps={this.defaultScrollViewProps}
```
Comment (colbymillerdev) [inline examples/ExampleOne.js]:
```suggestion
						scrollViewProps={defaultScrollViewProps}
```
The "this" keyword needs to be removed for this to work properly.

**Comment by colbymillerdev** [comment]
@DanSmit Hey Dan - I added a comment for a change that needs to be made before this can be merged. The "this" keyword isn't necessary when passing defaultScrollViewProps object to scrollViewProps in example files.

**Comment by colbymillerdev** [comment]
@DanSmit Hey Dan - Not sure if you saw my comment above yet. Did you want to make the change and push another commit, or do you want me to take care of it?

**Pull request merged**